tensorcircuit/translation.py

```diff
@@ -350,6 +350,20 @@ def qiskit2tc(
             # logger.warning(
             #     "qiskit to tc translation currently doesn't support reset instruction, just skipping"
             # )
+        elif gate_name == "barrier":
+            tc_circuit.barrier_instruction(*idx)
+        elif not hasattr(gate_info[0], "__array__"):
+            # an instruction containing a lot of gates.
+            # the condition is based on
+            # https://github.com/Qiskit/qiskit-terra/blob/2f5944d60140ceb6e30d5b891e8ffec735247ce9/qiskit/circuit/gate.py#L43
+            qiskit_circuit = gate_info[0].definition
+            if qiskit_circuit is None:
+                logger.warning(
+                    f"qiskit to tc translation doesn't support {gate_name} instruction, skipping"
+                )
+                continue
```

Comment:
no need for the continue line?

Reply:
If `qiskit_circuit` is None then the rest of the code block, which is converting from None to tc circuit, can not proceed. So IMO it's needed.

Reply:
yes, you are right, my bad, I am mix ``continue`` with ``pass``